tensorboard/plugins/hparams/hparams_demo.py

```diff
@@ -113,12 +145,12 @@ def run(logdir, session_id, hparams, group_name):
     hparams: A dictionary mapping an hyperparameter name to its value.
     group_name: an id for the session group this session belongs to.
   """
-  tf.compat.v1.reset_default_graph()
-  tf.compat.v1.set_random_seed(0)
+  tf.reset_default_graph()
```

Comment:
this file needs to be reverted to use `compat.v1`.  v2 does not have graph thus this API does not make sense.